packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -192,7 +192,16 @@ export interface BulkAddData {
     validationMsg?: ReactNode;
 }
 
+export interface BulkUpdateQueryInfoFormProps extends QueryInfoFormProps {
+    // the row ind to exclude for bulk update, row might be readonly or locked
+    excludeRowIdx?: number[];
+    onClickBulkUpdate?: (selected: Set<number>) => Promise<boolean>;
+    queryFilters?: { [key: string]: List<Filter.IFilter> };
```

Comment:
Same feedback as below. Be sure not to repeat properties from extended interfaces.